include/libdnf/repo/repo.hpp

```diff
@@ -100,6 +100,8 @@ class Repo {
     // @replaces libdnf:repo/Repo.hpp:method:Repo.setCallbacks(std::unique_ptr<RepoCallbacks> && callbacks)
     void set_callbacks(std::unique_ptr<libdnf::repo::RepoCallbacks> && callbacks);
 
+    std::unique_ptr<libdnf::repo::RepoCallbacks> & get_callbacks();
```

Comment:
Better to have a docstring there as it is public API.

Reply:
Oh! Thanks for the remainder.